pkg/pgmodel/ingestor/insert.go

```diff
@@ -385,13 +383,14 @@ func doInsert(conn pgxconn.PgxConn, reqs ...copyRequest) (err error) {
 
 	var affectedMetrics uint64
 	for _, numRows := range numRowsPerInsert {
-		ct, err := results.Exec()
+		var insertedRows int
+		err := results.QueryRow().Scan(&insertedRows)
 		if err != nil {
 			return err
 		}
-		if int64(numRows) != ct.RowsAffected() {
+		if int64(numRows) != int64(insertedRows) {
```

Comment:
Minor nit: maybe we can simplify these var types so we have to juggle less. I haven't looked at the code too deeply to check if its possible/easy.